javatests/arcs/showcase/imports/recipes/BUILD

```diff
@@ -0,0 +1,14 @@
+load(
+    "//third_party/java/arcs/build_defs:build_defs.bzl",
+    "arcs_kt_gen",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+arcs_kt_gen(
+    name = "gatherImports_recipe",
+    srcs = ["gatherImports.arcs"],
```

Comment:
The casing here is very unusual. Some suggestions: `gather_imports`, `gather_imports_recipe`, `GatherImports` (similarly updating the file name would be good).

Sorry to nitpick on naming so much, but it's good for examples to be as unsurprising as possible.

Reply:
As far as file names go: It looks like we use all different kinds of naming conventions for *.arcs files in the showcase folder. Should I standardize to one format? Either snake_case or TitleCase seems appropriate.

Reply:
I'll save this for a future PR.

Reply:
There was an initial push for TitleCase, but the users seem to default more naturally to snake_case, so I would revert the recommendation to it and update existing usages of TitleCase. FWIW.

Reply:
A follow up PR to make the naming consistent sounds really good to me (snake_case does seem to be more conventional, but I don't particularly mind)